MapboxNavigation/EndOfRouteViewController.swift

```diff
@@ -0,0 +1,163 @@
+import UIKit
+import MapboxDirections
+
+fileprivate enum ConstraintSpacing: CGFloat {
+    case closer = 16.0
+    case further = 45.0
+}
+
+class EndOfRouteViewController: UIViewController {
+
+    //MARK: - IBOutlets
+    @IBOutlet weak var primary: UILabel!
+    @IBOutlet weak var secondary: UILabel!
+    @IBOutlet weak var endNavigationButton: UIButton!
+    @IBOutlet weak var stars: RatingControl!
+    @IBOutlet weak var commentView: UITextView!
+    @IBOutlet weak var showCommentView: NSLayoutConstraint!
+    @IBOutlet weak var hideCommentView: NSLayoutConstraint!
+    @IBOutlet weak var ratingCommentsSpacing: NSLayoutConstraint!
+    
+    //MARK: - Properties
+    lazy var placeholder: String = NSLocalizedString("Add an optional comment here.", comment: "Comment Placeholder Text")
+    lazy var endNavigation: String = NSLocalizedString("End Navigation", comment: "End Navigation Button Text")
+    lazy var sendFeedback: String = NSLocalizedString("Send Feedback", comment: "Send Feedback Button Text")
+    
+    lazy var geocoder: CLGeocoder = CLGeocoder()
```

Comment:
Can you think of a way to allow the developer to substitute an alternative geocoder, such as MapboxGeocoder.swift?

Reply:
See https://github.com/mapbox/mapbox-navigation-ios/pull/848/files/5a6552fc3d2db1f4b22a63d4f090a0ae2b357608#r152429652